### PR TITLE
Beeware the underwear

### PR DESCRIPTION
**Pull request by nemvar**
## About The Pull Request

Adds normal sized bee socks, a tshirt with a bee on it, and yellow-black striped boxer shorts.

## Why It's Good For The Game

It's hip.

## Changelog
:cl:
add: New bee themed socks, tshirts and boxer shorts can now be chosen.
/:cl:

### CODE REVIEW

**Comment by AcapitalA** [comment]
:bee:

**Comment by wesoda25** [comment]
please stop this nemvar let the shitty meme die

**Comment by FantasticFwoosh** [comment]
I cannot bee-lieve you've made another bee clothing PR so soon.

**Comment by nicbn** [comment]
Well it was funny the first five hundred times, now it's just awkward

**Comment by HomocidlePancakes** [comment]
I think we have more underware options then any other game now. It’s time to stop.

**Review by ZeWaka — APPROVED** [review]

File: code/modules/mob/dead/new_player/sprite_accessories.dm [hunk 1]
```diff
@@ -1029,6 +1029,11 @@
 	icon_state = "male_white"
 	gender = MALE
 
+/datum/sprite_accessory/underwear/male_white
```
Comment (Ghommie) [inline code/modules/mob/dead/new_player/sprite_accessories.dm]:
```suggestion
/datum/sprite_accessory/underwear/male_bee
```

**Pull request closed without merging**